src/modules/loadPopup.js

```diff
@@ -0,0 +1,180 @@
+  const commentPopup = document.getElementById('comment-popup');
+  const reservationsPopup = document.getElementById('reservations-popup');
+
+  commentPopup.innerHTML = '';
+  reservationsPopup.innerHTML = '';
+
+  const popupContC = document.getElementById('popup-back');
+  const popupContR = document.getElementById('popup-back2');
+
+  // SHARED CONTENTS
+  const popupClose = document.createElement('span');
+  popupClose.className = 'popup-close';
+  popupClose.innerHTML = '<i class="fa-solid fa-xmark"></i>';
+
+  const popupimg = document.createElement('img');
+  popupimg.className = 'popup-img';
+  popupimg.src = movie.image.medium;
+
+  const popupTitle = document.createElement('h2');
+  popupTitle.textContent = movie.name;
+
+  const popupdes = document.createElement('div');
+  popupdes.innerHTML = movie.summary;
+  // END OF SHARED CONTENTS
+
+  // COMMENT SECTION CONTAINER
+  const popupCommentContainer = document.createElement('div');
+  popupCommentContainer.className = 'popup-comment-container';
+
+  const popupComment = document.createElement('div');
+  popupComment.dataset.popupIndex = index;
+  popupComment.className = 'comments';
+
+  const comentCounter = await getCommentCounter(index) || 0;
+  const popupCommentHeader = document.createElement('h2');
+  popupCommentHeader.innerHTML = `Comments (${comentCounter})`;
+
+  const popupCommentContent = document.createElement('div');
+
+  const popupCommentArray = await loadComments(index);
+
+  if (popupCommentArray.length) {
+    const popupCommentContentlist = document.createElement('ul');
+    popupCommentArray.forEach((comment) => {
+      const commentItem = document.createElement('li');
+
+      const date = document.createElement('span');
+      date.innerText = comment.creation_date;
+      const name = document.createElement('span');
+      name.innerText = `, ${comment.username}: `;
+      const text = document.createElement('span');
+      text.innerText = comment.comment;
+
+      commentItem.append(date, name, text);
+      popupCommentContentlist.append(commentItem);
+    });
+    popupCommentContent.append(popupCommentContentlist);
+  }
+
+  const popupFormComment = document.createElement('form');
+  popupFormComment.method = 'post';
+  popupFormComment.onsubmit = handleCommentSubmit;
+
+  const commentFormHead = document.createElement('h2');
+  commentFormHead.innerText = 'Write A Comment';
+  const popupInput1 = document.createElement('input');
+  popupInput1.name = 'name';
+  popupInput1.placeholder = 'Name';
+  const popupInput2 = document.createElement('input');
+  popupInput2.name = 'comment';
+  popupInput2.placeholder = 'Write a Comment...';
+  const popupSubmit = document.createElement('button');
+  popupSubmit.type = 'submit';
+  popupSubmit.innerText = 'Submit';
+
+  popupComment.append(popupCommentHeader, popupCommentContent);
+  popupFormComment.append(popupInput1, popupInput2, popupSubmit);
+  popupCommentContainer.append(popupComment, popupFormComment);
```

Comment:
- [ ] Good job working on the functionality 💪🏿, Kindly make your content aligned to the center just like the wireframe 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Comments](https://user-images.githubusercontent.com/69638013/181736618-139a9c2d-b4b3-4fa5-b8da-f079f2dc5ff0.png)  | ![Screenshot_364](https://user-images.githubusercontent.com/69638013/181736349-e3462dd4-269a-4138-a180-9219575f0b87.png)